website/blog/2023-05-14-databend-weekly-93.md

```diff
@@ -0,0 +1,126 @@
+---
+title: "This Week in Databend #93"
+date: 2023-05-14
+slug: 2023-05-14-databend-weekly
+tags: [databend, weekly]
+description: "Stay up to date with the latest weekly developments on Databend!"
+contributors:
+  - name: andylokandy
+  - name: b41sh
+  - name: BohuTANG
+  - name: Chasen-Zhang
+  - name: dantengsky
+  - name: DongHaowen
+  - name: Dousir9
+  - name: drmingdrmer
+  - name: everpcpc
+  - name: flaneur2020
+  - name: hantmac
+  - name: lichuang
+  - name: Mehrbod2002
+  - name: PsiACE
+  - name: SkyFan2002
+  - name: soyeric128
+  - name: sundy-li
+  - name: TCeason
+  - name: Xuanwo
+  - name: xudong963
+  - name: youngsofun
+  - name: zhang2014
+  - name: ZhiHanZ
+  - name: zhyass
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+> The `meta-upgrade-09` will not be available in the upcoming Databend release. If you are currently using version 0.9 or an earlier version, please utilize our upgrade tool which can be found at: [v1.1.33-nightly](https://github.com/datafuselabs/databend/releases/tag/v1.1.33-nightly).
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### Databend's Segment Caching Mechanism Now Boasts Improved Memory Usage
+
+Databend's segment caching mechanism has received a significant upgrade that reduces its memory usage to 1.5/1000 of the previous usage in a test scenario.
+
+The upgrade involves a different "representation" of cached segments, called `CompactSegmentInfo`. This presentation consists mainly of two components:
+
+- The decoded min/max indexes and other statistical information.
+- The undecoded (and compressed) raw bytes of block-metas.
+
+During segment pruning, if any segments are pruned, there is no need to decode the block-metas represented by raw bytes. If they are not pruned, then their raw bytes are decoded on-the-fly for block pruning and scanning purposes (and dropped if no longer needed).
+
+If you are interested in learning more, please check out the resources listed below.
+
+- [PR #11347 | refactor: hybrid segment cache](https://github.com/datafuselabs/databend/pull/11347)
+
+## Code Corner
+
+Discover some fascinating code snippets or projects that showcase our work or learning journey.
+
+### Bind `databend` into Python
+
+Databend now offers a Python binding that allows users to execute SQL queries using Python. This new feature eliminates the need for deploying instances.
```

Comment:
Databend now offers a Python binding that allows users to execute SQL queries against Databend using Python even without deploying a Databend instance.